File: 20_matrix_multiplication/19_alpha_numeric_palindrome/Palindrome.java
public class Palindrome {
    public static void main(String[] args) {
        String s = "I am :IronorI Ma, i";
        s = s.toLowerCase();
        s = s.replaceAll("[,:]", "");

        String rev = "";
        for(int i=0; i<s.length(); i++){
            rev = s.charAt(i) + rev;
        }

        if(s.equals(rev)){
            System.out.println("Palindrome");
        }
        else{
            System.out.println("Not a palindrome");
        }



    }
}

File: 07_palindrome/palindrome.py

word = "malayala"
# rev = ""

# for i in range(len(word)):
#     rev = word[i] + rev

# print(rev)

# if word == rev:
#     print("Palindrome")
# else:
#     print("Not a Palindrome")


for i in range(len(word)//2):
    if word[i] != word[-i-1]:
        print("Not a Palindrome")
        break
else:
    print("Palindrome")